runtime/src/fips.rs

```diff
@@ -0,0 +1,52 @@
+// Licensed under the Apache-2.0 license
+
+use caliptra_drivers::CaliptraError;
+use caliptra_drivers::CaliptraResult;
+use core::convert::TryFrom;
+
+use caliptra_drivers::KeyVault;
+use caliptra_registers::mbox::enums::MboxStatusE;
+/// FIPS module environment.
+pub struct FipsEnv<'a> {
+    pub key_vault: &'a mut KeyVault,
+}
+
+#[derive(PartialEq, Eq)]
+pub struct FipsModuleApi(pub u32);
+
+/// FIPS module commands.
+impl FipsModuleApi {
+    /// The status command.
+    pub const VERSION: Self = Self(0x4650_5652); // "FPVR"
+    /// The self-test command.
+    pub const SELF_TEST: Self = Self(0x4650_4C54); // "FPST"
+    /// The shutdown command.
+    pub const SHUTDOWN: Self = Self(0x4650_5344); // "FPSD"
+}
```

Comment:
Could we just group these with the regular mailbox commands in lib.rs? They can still be implemented in fips.rs

Reply:
Sure.